src/apps/vpn/commands/commandui.cpp

```diff
@@ -63,7 +63,7 @@
 #  include "platforms/macos/macosutils.h"
 #endif
 
-#ifdef MZ_ANDROID`
```

Comment:
unrelated to the issue but pretty sure this was a typo

Reply:
Already fixed. Probably you need to rebase this PR.